analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -131,7 +132,7 @@ class Analyzer(private val config: AnalyzerConfiguration) {
         managedFiles: Map<PackageManager, List<File>>,
         packageCurationsFile: File?
     ): AnalyzerResult {
-        val dispatcher = Executors.newFixedThreadPool(5).asCoroutineDispatcher()
+        val dispatcher = Executors.newFixedThreadPool(5, NamedThreadFactory("analyzer")).asCoroutineDispatcher()
```

Comment:
Should we maybe use `TOOL_NAME` here (and for the scanner)?

Reply:
Done. Use the storage and scanner names in `LocalScanner` now.